rulesets/src/url-structure.ruleset.yml

```diff
@@ -79,8 +79,8 @@ rules:
   sps-paths-kebab-case:
     message: "A resource containing multiple words MUST be separated using kebab-case (lower case and separated with hyphens)."
     severity: error
-    given: $.paths.*~
-    then:
+    given: $.paths[?(/^((?!_webhooks).)*$/i.test(@property))]~
```

Comment:
![image](https://user-images.githubusercontent.com/287161/225672146-1832128e-a447-4ceb-b76b-2297bf391d9f.png)

Reply:
"DEVILRY" is a good term for what this is...haha... this is a regex testing pattern with negative lookahead to exclude this rule if the path includes "_webhooks"